hera_qm/ant_metrics.py

```diff
@@ -59,235 +63,194 @@ def get_ant_metrics_dict():
-        Default is False.
+        Defaults None means all visibility polarizations are used.
 
     Returns
     -------
-    meanMetrics : dict
-        Dictionary indexed by (ant, antpol) of the modified z-score of the
-        mean of the absolute value of all visibilities associated with an antenna.
+    per_ant_mean_corr_metrics : dict
+        Dictionary indexed by (ant, antpol) of the
+        mean of correlation value associated with an antenna.
         Very small or very large numbers are probably bad antennas.
 
     """
+
     from hera_cal.utils import split_pol, split_bl
 
     # figure out which antennas match pols and and are not in xants
     if pols is not None:
-        antpols = set([ap for bl in abs_vis_stats for ap in split_pol(bl[2])
+        antpols = set([ap for bl in corr_stats for ap in split_pol(bl[2])
                        if ((pols is None) or (bl[2] in pols))])
     ants = set()
-    for bl in abs_vis_stats:
+    for bl in corr_stats:
         for ant in split_bl(bl):
             if (ant not in xants) and (ant[0] not in xants):
                 if (pols is None) or (ant[1] in antpols):
                     ants.add(ant)
 
-    # assign visibility means to each antenna in the baseline
-    per_ant_means = {ant: [] for ant in ants}
-    for bl, vis_mean in abs_vis_stats.items():
+    # assign correlation metrics to each antenna in the baseline
+    per_ant_corrs = {ant: [] for ant in ants}
+    for bl, corr_mean in corr_stats.items():
         if bl[0] == bl[1]:
             continue  # ignore autocorrelations
         if (pols is None) or (bl[2] in pols):
-            for ant in split_bl(bl):
-                if ant in ants:
-                    per_ant_means[ant].append(vis_mean)
-    per_ant_means = {ant: np.nanmean(per_ant_means[ant]) for ant in ants}
+            if split_bl(bl)[0] in ants and split_bl(bl)[1] in ants:
+                for ant in split_bl(bl):
+                    per_ant_corrs[ant].append(corr_mean)
+    per_ant_mean_corr_metrics = {ant: np.nanmean(per_ant_corrs[ant]) for ant in ants}
 
-    if rawMetric:
-        return per_ant_means
-    else:
-        return per_antenna_modified_z_scores(per_ant_means)
+    return per_ant_mean_corr_metrics
 
 
-def antpol_metric_sum_ratio(cross_metrics, same_metrics):
-    """Compute ratio of two metrics summed over polarizations.
+def corr_cross_pol_metrics(corr_stats, xants=[]):
+    """Calculate the differences in the correlation metric between polarizations.
 
-    Takes the ratio of two antenna metrics, summed over both polarizations, and creates
-    a new antenna metric with the same value in both polarizations for each antenna.
-    For example, if we're looking a metric for antenna 1, m1[pol], this would be:
-    (m1['en'] + m1['ne']) / (m1['ee'] + m1['nn']) which would be inserted in both
-    cross_pol_ratio[1, 'Jee'] and cross_pol_ratio[1, 'Jnn'].
+    The four polarization combinations are xx-xy, yy-xy, xx-yx, and yy-yx. An
```

Comment:
I'm a bit confused by this comment since it seems redundant to have the inputs and outputs listed twice - I tried to make the docs for this function clearer, so let me know if what I did is sufficient.

Reply:
Maybe this was a problem with how the diff was rendered. Looking at the actual file I can see docstring looks complete.